Security/TwoFactor/AuthenticationContextInterface.php

```diff
@@ -63,4 +63,11 @@ public function isAuthenticated();
      * @param bool $authenticated
      */
     public function setAuthenticated($authenticated);
+
+    /**
+     * Is authentication request.
+     *
+     * @return bool
+     */
+    public function isAuthenticationTry();
```

Comment:
Yes, it is meant to be an extension point for users who need additional functonallity in the AuthenticationContext. See #55.